vowpalwabbit/core/CMakeLists.txt

```
@@ -354,6 +356,7 @@ vw_add_library(
     ${ENABLE_INSTALL_OPTION}
 )
 
+target_include_directories(vw_core PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
```

`CMAKE_CURRENT_LIST_DIR` is usually what you expect when you use a variable like this. `CMAKE_CURRENT_SOURCE_DIR` works but if this file was included then I think it would behave differently
```suggestion
target_include_directories(vw_core PRIVATE ${CMAKE_CURRENT_LIST_DIR}/src)
```